notion/README.md

```diff
@@ -0,0 +1,35 @@
+# Agent Check: Notion
+
+## Overview
+
+Notion is a Connected Workspace. It's where modern teams create and share docs, take notes, manage projects, and organize knowledge, all in one place. Install Notion's integration with Datadog to manage and monitor your workspace activity in [Datadog Cloud SIEM][1]. You can import your workspace's audit logs for real-time monitoring, alerting, and analysis. From there, you can detect and investigate potential security issues, suspicious behavior, and troubleshoot access with confidence and ease.
+
+## Setup
+
+Datadog setup
+Open Datadog and go to Integrations > Integrations.
+
+Search for "Notion".
+
+Go to the Configuration tab and click Install Integration at the bottom.
+
+In Notion navigate to Setting & Members > Connections > Workspace Connections > +Add Connection > Datadog. 
+
+Notion will prompt you to follow a series of OAuth steps to authorize the integration with Datadog.
+
+Once connected, Notion will start sending near real-time data to Datadog.
+
+## Uninstallation
+In Notion navigate to Setting & Members > Connections > Workspace Connections > ... next to Datadog > Disconnect
```

Comment:
Please include the oauth uninstallation as seen [here](https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/#publish-the-oauth-client)